Gruntfile.js

```diff
@@ -144,8 +146,12 @@ module.exports = function(grunt) {
 
     grunt.config.set("gh-release.release-notes", releaseNotes);
 
-    grunt.file.write(dest, "" + header + contents);
-    grunt.log.ok("Changelog updated, and release notes extracted.");
+    if (dryRun) {
+      grunt.log.ok('changelog (dry): Prepending header to ' + dest);
```

Comment:
Sure, but logging the release notes to the console might generate quite a bit of output if you have a large release.